### PR TITLE
DRILL-7125: REFRESH TABLE METADATA fails after upgrade from Drill 1.1…

### PR DESCRIPTION
**Pull request by sohami**
…3.0 to Drill 1.15.0

### CODE REVIEW

File: exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/metadata/Metadata.java [hunk 1]
```diff
@@ -675,32 +699,28 @@ private boolean tableModified(List<Path> directories, Path metaFilePath, Path pa
...
     }
+    return logAndStopTimer(isModified, timer, numDirs);
+  }
+
+  private boolean logAndStopTimer(boolean isModified, Stopwatch timer, int numDirectories) {
     if (timer != null) {
-      logger.debug("No directories were modified. Took {} ms to check modification time of {} directories",
-          timer.elapsed(TimeUnit.MILLISECONDS), numDirs);
+      logger.debug("Directory {} was modified. Took {} ms to check modification time of {} directories",
+        isModified, timer.elapsed(TimeUnit.MILLISECONDS), numDirectories);
```
Comment (amansinha100) [inline exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/metadata/Metadata.java]: It is actually useful to have a log message even for the case when no directories were modified..that way we can account for the elapsed time.  You can preserve the prior log message conditional based on the `isModified` flag.

Comment (sohami) [reply exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/metadata/Metadata.java]: Thanks for catching this. Somehow I missed logging was done in both cases. Have made the change.

**Comment by arina-ielchiieva** [comment]
+1

**Comment by sohami** [comment]
Thanks for the review. Squashed and rebased.

**Pull request merged**